os/include/tinyara/fs/ioctl.h

```diff
@@ -153,6 +153,11 @@
 										 *      (Guaranteed to persist while the file
 										 *      is open).
 										 */
+#define FIONBIO         _FIOC(0x000b)     /* IN:  Boolean option takes an
```

Comment:
How about moving this under `FIONREAD` because of order?

Reply:
done